Makefile

```diff
@@ -42,12 +46,15 @@ poetry:
 	$(call _run_interactive, //bin/sh -c "poetry shell")
 
 unit-test:
-	$(call _run_bare, poetry run pytest -vv)
+	$(call _run_bare, poetry run pytest -vv ${EXTRA_PYTEST_FLAGS})
 
 test: build unit-test lint
 
+test-with-coverage:
+	$(MAKE) WITH_COVERAGE=YES test
+
 test-watch: build
-	$(call _run_interactive, poetry run pytest-watch -- -vv)
+	$(call _run_interactive, poetry run pytest-watch -vv)
```

Comment:
Is this correct? The `--` is to separate `pytest-watch` args from `pytest` args.